rules/web/web_cve_2022_27925_exploit.yml

```diff
@@ -29,7 +30,7 @@ detection:
     selection_shell:
         cs-uri|contains: '/zimbraAdmin/'
         cs-uri|endswith: '.jsp'
```

Comment:
In the docs you linked  `cs-uri` doesn't exist

Reply:
`cs-uri-query` 👍